File: DotNet/Apps/ConfMgrGUI/OpsTrackingModel/src/OpsTrackingModel/InboundDocsView.java
package OpsTrackingModel;

import java.io.Serializable;
import java.util.Date;

//import com.gigaspaces.annotation.pojo.SpaceClass;
//import com.gigaspaces.annotation.pojo.SpaceId;
//import com.gigaspaces.annotation.pojo.SpaceProperty;
//import com.gigaspaces.annotation.pojo.SpaceRouting;

//@SpaceClass
public class InboundDocsView implements Serializable{

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
    private long _id = -1;
    private long _unresolvedCount = -1;
    private String _callerRef;
    private String _sentTo;
    private Date _rcvdTs = null;
    private String _fileName;
    private String _sender;
    private String _cmt;
    private String _docStatusCode;
    private String _hasAutoAsctedFlag;
    private String _tradeIds;
    private String _bookmarkFlag;
    private String _bookmarkUser;
    private String _ignoreFlag;
    private String _ignoredUser;
    private String _commentFlag;
    private String _commentUser;
    private String _userComments;
    private String _mappedCptySn;
    private String _procFlag;
	
//	@SpaceId    //	Sets this field (ID) to be used in the construction
  //  @SpaceRouting
 //   @SpaceProperty(nullValue="-1")
    public long get_id() {
 		return _id;
 	}
 	
    public void set_id(long _id) {
 		this._id = _id;
 	}
  //  @SpaceProperty(nullValue="-1")
 	public long get_unresolvedCount() {
 		return _unresolvedCount;
 	}

    public void set_unresolvedCount(long count) {
 		_unresolvedCount = count;
 	}
 	
 	public String get_callerRef() {
 		return _callerRef;
 	}
 	
 	public void set_callerRef(String ref) {
 		_callerRef = ref;
 	}
 	
 	public String get_sentTo() {
 		return _sentTo;
 	}
 	
 	public void set_sentTo(String to) {
 		_sentTo = to;
 	}
 	
 	public Date get_rcvdTs() {
 		return _rcvdTs;
 	}
 	
 	public void set_rcvdTs(Date ts) {
 		_rcvdTs = ts;
 	}
 	
 	public String get_fileName() {
 		return _fileName;
 	}
 	
 	public void set_fileName(String name) {
 		_fileName = name;
 	}
 	
 	public String get_sender() {
 		return _sender;
 	}
 	
 	public void set_sender(String _sender) {
 		this._sender = _sender;
 	}
 	
 	public String get_cmt() {
 		return _cmt;
 	}
 	
 	public void set_cmt(String _cmt) {
 		this._cmt = _cmt;
 	}
 	
 	public String get_docStatusCode() {
 		return _docStatusCode;
 	}
 	
 	public void set_docStatusCode(String statusCode) {
 		_docStatusCode = statusCode;
 	}
 	
 	public String get_hasAutoAsctedFlag() {
 		return _hasAutoAsctedFlag;
 	}
 	
 	public void set_hasAutoAsctedFlag(String autoAsctedFlag) {
 		_hasAutoAsctedFlag = autoAsctedFlag;
 	}
 	
 	public String get_tradeIds() {
 		return _tradeIds;
 	}
 	
 	public void set_tradeIds(String ids) {
 		_tradeIds = ids;
 	}
 	
 	public String get_bookmarkFlag() {
 		return _bookmarkFlag;
 	}
 	
 	public void set_bookmarkFlag(String flag) {
 		_bookmarkFlag = flag;
 	}
 	
 	public String get_bookmarkUser() {
 		return _bookmarkUser;
 	}
 	
 	public void set_bookmarkUser(String user) {
 		_bookmarkUser = user;
 	}
 	
 	public String get_ignoreFlag() {
 		return _ignoreFlag;
 	}
 	
 	public void set_ignoreFlag(String flag) {
 		_ignoreFlag = flag;
 	}
 	
 	public String get_ignoredUser() {
 		return _ignoredUser;
 	}
 	
 	public void set_ignoredUser(String user) {
 		_ignoredUser = user;
 	}
 	
 	public String get_commentFlag() {
 		return _commentFlag;
 	}
 	
 	public void set_commentFlag(String flag) {
 		_commentFlag = flag;
 	}
 	
 	public String get_commentUser() {
 		return _commentUser;
 	}
 	
 	public void set_commentUser(String user) {
 		_commentUser = user;
 	}
 	
 	public String get_userComments() {
 		return _userComments;
 	}
 	
 	public void set_userComments(String comments) {
 		_userComments = comments;
 	}
 	
 	public String get_mappedCptySn() {
 		return _mappedCptySn;
 	}
 	
 	public void set_mappedCptySn(String cptySn) {
 		_mappedCptySn = cptySn;
 	}
 	
 	public String get_procFlag() {
		return _procFlag;
	}

	public void set_procFlag(String procflag) {
		_procFlag = procflag;
	}
	
	@Override
 	public String toString(){
 		return _id+"|"+_unresolvedCount+"|"+_callerRef+"|"+_sentTo+"|"+_rcvdTs+"|"+_fileName+"|"+_sender+"|"+_cmt+"|"+_docStatusCode+"|"+_hasAutoAsctedFlag+"|"+_tradeIds+"|"+_bookmarkFlag+"|"+_bookmarkUser+"|"+_ignoreFlag+"|"+_ignoredUser+"|"+_commentFlag+"|"+_commentUser+"|"+_userComments+"|"+_mappedCptySn+"|"+_procFlag;
 	}
 	
}

File: DotNet/Apps/ConfMgrGUI/OpsTrackingModel/InboundDocsView.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace OpsTrackingModel
{
    [System.Serializable]
    public class InboundDocsView:IOpsDataObj
    {
        private const Int32 _nullNumberValue = -1;
        private const string _nullDateTime = "1900-01-01T12:00:00";

        private System.Int32 _id;

        private System.Int32 _unresolvedCount;

        private string _callerRef;
        private string _sentTo;

        private System.DateTime _rcvdTs;

        private string _fileName;
        private string _sender;
        private string _cmt;
        private string _docStatusCode;
        private string _hasAutoAsctedFlag;
        private string _tradeIds;
        private string _bookmarkFlag;
        private string _bookmarkUser;
        private string _ignoreFlag;
        private string _ignoredUser;
        private string _commentFlag;
        private string _commentUser;
        private string _userComments;
        private string _mappedCptySn;
        private string _procFlag;

        public string ProcFlag
        {
            get { return _procFlag; }
            set { _procFlag = value; }
        }

        public string MappedCptySn
        {
            get { return _mappedCptySn; }
            set { _mappedCptySn = value; }
        }

        public string UserComments
        {
            get { return _userComments; }
            set { _userComments = value; }
        }

        public string CommentUser
        {
            get { return _commentUser; }
            set { _commentUser = value; }
        }

        public string CommentFlag
        {
            get { return _commentFlag; }
            set { _commentFlag = value; }
        }

        public string IgnoredUser
        {
            get { return _ignoredUser; }
            set { _ignoredUser = value; }
        }

        public string IgnoreFlag
        {
            get { return _ignoreFlag; }
            set { _ignoreFlag = value; }
        }

        public string BookmarkUser
        {
            get { return _bookmarkUser; }
            set { _bookmarkUser = value; }
        }

        public string BookmarkFlag
        {
            get { return _bookmarkFlag; }
            set { _bookmarkFlag = value; }
        }

        public string TradeIds
        {
            get { return _tradeIds; }
            set { _tradeIds = value; }
        }

        public string Tradeids // added for DB loading of database objects.  Set field is not recognized.  Correct fix should be to modify the SQL.  Refactor is an issue though.
        {
            set { TradeIds = value; }
        }

        public string HasAutoAsctedFlag
        {
            get { return _hasAutoAsctedFlag; }
            set { _hasAutoAsctedFlag = value; }
        }

        public string DocStatusCode
        {
            get { return _docStatusCode; }
            set { _docStatusCode = value; }
        }

        public string Cmt
        {
            get { return _cmt; }
            set { _cmt = value; }
        }

        public string Sender
        {
            get { return _sender; }
            set { _sender = value; }
        }

        public string FileName
        {
            get { return _fileName; }
            set { _fileName = value; }
        }

        public System.DateTime RcvdTs
        {
            get { return _rcvdTs; }
            set { _rcvdTs = value; }
        }

        public string SentTo
        {
            get { return _sentTo; }
            set { _sentTo = value; }
        }

        public string CallerRef
        {
            get { return _callerRef; }
            set { _callerRef = value; }
        }

        public System.Int32 UnresolvedCount
        {
            get { return _unresolvedCount; }
            set { _unresolvedCount = value; }
        }

        public System.Int32 Unresolvedcount // added for DB loading of database objects.  Set field is not recognized.  Correct fix should be to modify the SQL.  Refactor is an issue though.
        {
            set { UnresolvedCount = value; }
        }

        public System.Int32 Id
        {
            get { return _id; }
            set { _id = value; }
        }

        public InboundDocsView()
        {
            this._id = InboundDocsView._nullNumberValue;
            this._unresolvedCount = InboundDocsView._nullNumberValue;
            this._callerRef = null;
            this._sentTo = null;
            this._rcvdTs = System.DateTime.Parse(InboundDocsView._nullDateTime);
            this._fileName = null;
            this._sender = null;
            this._cmt = null;
            this._docStatusCode = null;
            this._hasAutoAsctedFlag = null;
            this._tradeIds = null;
            this._bookmarkFlag = null;
            this._bookmarkUser = null;
            this._ignoreFlag = null;
            this._ignoredUser = null;
            this._commentFlag = null;
            this._commentUser = null;
            this._userComments = null;
            this._mappedCptySn = null;
            this._procFlag = "Y";
        }

        public string getSelect()
        {
            return Properties.Settings.Default.InboundDocsView;
        }
    }
}
